lib/sqewer/version.rb

```diff
@@ -1,3 +1,3 @@
 module Sqewer
-  VERSION = '8.0.3'
+  VERSION = '8.0.4'
```

Comment:
I wouldn't consider this as a patch, given some may be relying on `Ks` in their projects 🤔

Reply:
One should not depend on transient dependencies, but agree, raised minor

Reply:
updated

Reply:
Ideally not, but many examples used to use `Ks`, so then it's a slightly different story 🤷‍♂️